canonical/canonical.com
# Add GIA candidate feedback download link to Candidate page

## Done

- Add functionality to allow a candidate to download their assessment documents from the candidate page

## QA

- Open the demo
- Go to /careers/application/{token} - [tokens in different states](https://pastebin.canonical.com/p/xSw9w6hhmD/)
- Check that in the Assessment stage there is a request button when appropriate
- Click the button and try to enter a random email and check the validation works
- Use the candidate's email address and submit the form
- Check that this is successful and provides one or more reports to open

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3015
